microstoragetest/storage.go

```diff
@@ -0,0 +1,18 @@
+package microstoragetest
+
+import (
+	"fmt"
+
+	"github.com/giantswarm/microstorage"
+	"github.com/giantswarm/microstorage/memory"
+)
+
+// New creates a configured storage ready to be used in tests.
+func New() microstorage.Storage {
```

Comment:
In case a method panics eventually the golang convention for function naming is using the `Must` prefix.